src/Network/GRPC/Server/Settings.hs

```diff
@@ -0,0 +1,39 @@
+module Network.GRPC.Server.Settings
+  ( ServerSettings(..)
+  , defaultServerSettings
+  ) where
+
+import Data.Default
+
+import Network.GRPC.Common.HTTP2Settings
+
+-- | Settings affecting the server's concurrency and HTTP\/2 parameters
+data ServerSettings = ServerSettings {
+      -- | Number of threads that will be spawned to process incoming frames
+      -- on the currently active HTTP\/2 streams
+      --
+      -- This setting is specific to the
+      -- [http2](https://hackage.haskell.org/package/http2) package's
+      -- implementation of the HTTP\/2 specification.
+      --
+      -- __Note__: If a lower 'serverInitialConnectionWindowSize' is desired, the
+      -- number of workers should be increased to avoid a potential HTTP\/2
+      -- control flow deadlock.
+      serverNumberOfWorkers :: Word
+
+      -- | HTTP\/2 settings
+    , serverHTTP2Settings :: HTTP2Settings
+    }
+  deriving (Show)
```

Comment:
Does this really have to be a separate record that we pass in separate from the `ServerConfig`? I would prefer if we could just add these two values there, seems simpler.

Reply:
Yes, I will combine these!